apps/filebrowser/src/filebrowser/templates/listdir.mako

```diff
@@ -61,7 +61,7 @@ ${ fb_components.menubar() }
         <div class="btn-toolbar" style="display: inline; vertical-align: middle">
           <div id="ch-dropdown" class="btn-group" style="vertical-align: middle">
             <button class="btn dropdown-toggle" title="${_('Actions')}" data-toggle="dropdown"
-            data-bind="visible: !inTrash(), enable: selectedFiles().length > 0 && ((!isS3() && !isABFS() && !isOFS()) || (isS3() && !isS3Root()) || (isABFS() && !isABFSRoot()) || (isOFS() && !(isOFSRoot() || isOFSServiceID() || isOFSVol())))">
+            data-bind="visible: !inTrash(), enable: selectedFiles().length > 0 && ((!isS3() && !isGS() && !isABFS() && !isOFS()) || (isS3() && !isS3Root()) || (isGS() && !isGSRoot()) || (isABFS() && !isABFSRoot()) || (isOFS() && !(isOFSRoot() || isOFSServiceID() || isOFSVol())))">
```

Comment:
If would be nice to place these long conditions as a computed observable in the FileBrowserModel in apps/filebrowser/src/filebrowser/templates/listdir_components.mako instead

Reply:
Would need some help with that since I am a bit unaware on the ko observables. Does it makes sense to do that in a separate PR and leaving it here like it was done for other FS?

In long term, we have to get rid of mako anyway is what I am thinking.

Reply:
Yes to the longterm option